index.js

```diff
@@ -29,8 +29,9 @@ bitcore.errors = require('./lib/errors');
 
 // main bitcoin library
 bitcore.Address = require('./lib/address');
-bitcore.Block = require('./lib/block');
-bitcore.BlockHeader = require('./lib/blockheader');
+bitcore.Block = require('./lib/block/block');
```

Comment:
Yup I thought of that when making the change but wasn't sure which would be preferable.  I'll make that change.  Thanks.